playground/src/index.js

```diff
@@ -16,7 +16,7 @@ import {
 	BlockList,
 	WritingFlow,
 	ObserveTyping,
-	LinkControl,
+	__experimentalLinkControl,
```

Comment:
You can nuke this from your PR as we won't be merging _any_ changes to the Playground.

Reply:
If it's still in your branch (the one this PR targets), I'd keep it so things keep working. If it was already removed, I can rebase and delete it here too.

Reply:
Let's leave it and remove from the main canonical PR.